README.org

```diff
@@ -453,6 +456,79 @@ For a more general index related to all-things Emacs, use [[https://github.com/e
 
     Also handles macros, functions defined through macros and let-bound variables.
 
+    If you want to jump to symbols in files which aren't loaded in your Emacs you can fallback to the more general [[https://github.com/jacktasia/dumb-jump][dumb-jump]] package.
```

Comment:
This sentence should appear before the elisp-def sub-section

Reply:
Done